test/common/access_log/access_log_formatter_test.cc

```diff
@@ -382,6 +382,210 @@ TEST(AccessLogFormatterTest, startTimeFormatter) {
   }
 }
 
+TEST(AccessLogFormatterTest, JsonFormatterTest) {
```

Comment:
Can you break these into seperate `TEST`s. It's best to have distinct tests to control reruns and have finer grained visibility into specific failures. Also, can you add a one line `//` description above each?

Reply:
Sure! Was following the style of the file, will break them up now